### PR TITLE
MCSC-961 Scheme Credit Consent Content

### PR DESCRIPTION
**Pull request by TJSarno**

### CODE REVIEW

File: app/models/SchemeCreditConsent.scala [hunk 1]
```diff
@@ -16,31 +16,16 @@
...
-        content = Text(messages(s"schemeCreditConsent.${value.toString}")),
-        fieldId = "value",
-        index = index,
-        value = value.toString
-      )
-    }
-
-  implicit val enumerable: Enumerable[SchemeCreditConsent] =
-    Enumerable(values.map(v => v.toString -> v): _*)
+  def saveConsent(userAnswers: UserAnswers, boolean: Boolean): UserAnswers =
```
Comment (MichaelAAbbott) [inline app/models/SchemeCreditConsent.scala:28]: Delete this method, it is not used anywhere

File: conf/messages.en
Comment (MichaelAAbbott) [inline conf/messages.en:1]: Welsh place holders please :)

File: test/controllers/SchemeCreditConsentControllerSpec.scala
Comment (MichaelAAbbott) [inline test/controllers/SchemeCreditConsentControllerSpec.scala:1]: Needs unit tests to check the user answer is being set

Comment (MichaelAAbbott) [reply test/controllers/SchemeCreditConsentControllerSpec.scala:1]: Also needs unit tests to check for redirection to the next page

**Review by MichaelAAbbott — CHANGES_REQUESTED** [review]

**Review by bharti5verma — APPROVED** [review]

**Pull request closed without merging**